setup.py

```diff
@@ -12,7 +13,7 @@
 # make sure that submodules are up-to-date,
 # it is a common error to forget this when
 # switching between development branches
-subp.check_call(["git", "submodule", "update"])
+# subp.check_call(["git", "submodule", "update"])
 
```

Comment:
This is fixed in the Pythia8 branch, please accept the conflicting change when Pythia8 branch is merged.

Reply:
I see. I will.